src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -294,6 +294,8 @@ public static void main(String[] args) {
    */
   private final Set<String> typesUsed = new LinkedHashSet<>();
 
+  private Map<String, String> importRenameMap = new HashMap<>();
```

Comment:
guess what I'd like to see on this field? Starts with a `d` :-)

Reply:
Done.